app/views/asp/schoolings/index.html.haml

```
@@ -120,5 +120,9 @@
                 %td= number_to_currency pfmp.amount
               %tr
                 %td Date d'envoi du flux
-                %td= l payment_request.asp_request.sent_at, format: :long
+                %td
+                  - if payment_request.asp_request.sent_at
+                    = l payment_request.asp_request.sent_at, format: :long
+                  - else
+                    Pas de date d'envoi renseignée
```

on aura toujours une date d'envoi renseignée s'il y a une `asp_request`

---

c'est exactement pour ça qu'on ne va pas accepter un cas de "qui sait" et appliquer le principe du "fail early" pour être sûr qu'on attrape ces bugs.

---

Je suis pas d'accord. En terme de service, ce serait bête d'empêcher l'asp de voir cette page et de faire leur contrôle juste par ce qu'on a pas de date d'envoi.